text/setup.py

```diff
@@ -41,6 +41,8 @@
     f'autogluon.features=={version}',
     f'autogluon.common=={version}',
     'autogluon-contrib-nlp==0.0.1b20220208',
+    'nlpaug',
```

Comment:
1. missing min and max versions
2. How heavy is this package? Is it something we want as a required install?

Reply:
nlpaug is about 2.2M, and it is used during text augmentation.

Reply:
https://pypi.org/project/nlpaug/ . The latest version seems to be 1.1.10.

Reply:
Yes, I made another commit specify the version.